bundles/org.openhab.binding.pulseaudio/src/main/resources/OH-INF/thing/sink.xml

```diff
@@ -33,6 +33,15 @@
 				<description>Default Port to allocate for use by module-simple-protocol-tcp on the pulseaudio server</description>
 				<default>4711</default>
 			</parameter>
+			<parameter name="simpleProtocolSinkIdleTimeout" type="integer" required="false">
+				<label>Idle Timeout</label>
+				<description>Timeout in ms after which the connection will be closed when no stream is running.
+					This ensures that
```

Comment:
These line breaks look a bit weird here. Could you clean this up?

Reply:
Sure. The linter complained about it and I ran `mvn spotless:apply` which resulted in this weirdly formatted code. I tried to make it a bit nicer without the linter going nuts.